docs/modules/integrate/pages/cdc-connectors.adoc

```diff
@@ -79,20 +187,28 @@ For example, a sink mapping CDC data to a `Customer` class and
 maintaining a map view of latest known email addresses per customer
 (identified by ID) would look like this:
 
-```java
+[source,java]
+----
 Pipeline p = Pipeline.create();
 p.readFrom(source)
  .withoutTimestamps()
  .writeTo(CdcSinks.map("customers",
     r -> r.key().toMap().get("id"),
     r -> r.value().toObject(Customer.class).email));
-```
+----
 
 [NOTE]
 ====
 The key and value functions have certain limitations. They can be used to map only to objects which the Hazelcast member can deserialize, which unfortunately doesn't include user code submitted as a part of the job. So in the above example it's OK to have `String` email values, but we wouldn't be able to use `Customer` directly.
 
 If user code has to be used, then the problem can be solved with the help of the User Code Deployment feature. Example configs for that can be seen in our xref:pipelines:cdc-join.adoc#7-start-hazelcast-jet[CDC Join tutorial].
+====
+
+== Migration Guide
+
+Hazelcast {open-source-product-name} has Debezium CDC connector, however it is based on older version of Debezium.
+While migrating to the new connector should be very simple, there are some things users should remember:
 
-Although User Code Deployment has been deprecated, the replacement User Code Namespaces feature does not yet support Jet jobs or pipelines. For now, continue to use the User Code Deployment solution in this scenario. 
-====
\ No newline at end of file
+ * Package was changed from `com.hazelcast.jet.cdc` to `com.hazelcast.enterprise.jet.cdc`
```

Comment:
```suggestion
 * Package renamed from `com.hazelcast.jet.cdc` to `com.hazelcast.enterprise.jet.cdc`
```
Is it just a rename or a change of package?

Reply:
Hard to say if it makes a difference in this context - old connector has old package new, new connector has new package name (changed/renamed)

Reply:
Yeah, I thought that - probably better with:
```suggestion
 * You should use the `com.hazelcast.enterprise.jet.cdc` package instead of `com.hazelcast.jet.cdc`.
```